Channels/Insider/release_notes.md

```diff
@@ -1,24 +1,39 @@
-## September 27 2022 Insider Release ([v1.1.2089.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.2089.01))
+## January 30 2023 Insider Release ([v1.1.2213.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.2213.01))
 
-Welcome to the September 27 2022 Insider release of Accessibility Insights for Windows.
+Welcome to the January 30 2023 Insider release of Accessibility Insights for Windows.
 
-Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.2089.01/AccessibilityInsights.msi
+Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.2213.01/AccessibilityInsights.msi
 
 Documentation Link: https://accessibilityinsights.io/docs/en/windows/overview
 
 ### Highlights
 
+- [New Features](#new-features)
 - [Rule Updates](#rule-updates)
 - [Bug Fixes](#bug-fixes)
 
+#### New Features
+
+- Improve audio feedback in the product
```

Comment:
Suggestion, lets start this section with:

We have added new features to improve the user experience for users who are blind:

* Sounds options...
* new sounds ...
* sounds for scanning is now unique..
* discoverability of keystrokes..
* display MSAA..